FreshRSS/FreshRSS
# fix: unread icons in Ansum/Mapco theme

Closes #4645 

Now (Mapco theme):
![grafik](https://user-images.githubusercontent.com/1645099/192573584-e9395d52-0aac-4aa3-b9ff-30474db04a8e.png)


Changes proposed in this pull request:

- svg icon "unread.svg" now in square size für Ansum/Mapco theme


How to test the feature manually:

1. use Mapco theme
2. use Ansum theme
3. see the unread button


Pull request checklist:
